Libs/MRML/Core/CMakeLists.txt

```diff
@@ -63,8 +63,9 @@ endif()
 #
 # vtkAddon
 #
-if (Slicer_USE_SYSTEM_vtkAddon)
+if(NOT Slicer_BUILD_vtkAddon)
   find_package(vtkAddon REQUIRED)
+  INCLUDE(${vtkAddon_DIR}/vtkAddonTargets.cmake)
```

Comment:
Instead of including the file here, I improved vtkAddon to take care of it. See https://github.com/Slicer/vtkAddon/pull/5